user-interface/src/staff-assignment/screen/StaffAssignmentScreen.test.tsx

```diff
@@ -31,16 +37,11 @@ describe('StaffAssignmentScreen', () => {
     };
 
     const SearchResults = vi
-      .spyOn(searchResultsModule, 'SearchResults')
```

Comment:
when looking at the coverage, this was not needed as part of the coverage for StaffAssignmentScreen.